src/execution/operator/aggregate/physical_window.cpp

```diff
@@ -390,7 +392,7 @@ static idx_t FindPrevStart(const BitArray<W> &mask, const idx_t l, idx_t r) {
 	return l;
 }
 
-static void MaterializeExpressions(Expression **exprs, idx_t expr_count, ChunkCollection &input,
+static void MaterializeExpressions(Expression **exprs, idx_t expr_count, ChunkCollection &input, // NOLINT
```

Comment:
Are all these NOLINT's required?

Reply:
Well my editor complains (via our clangd settings) that there are two arguments that can be confused. I don't know why clang-tidy lets them through.

Reply:
Perhaps a different version of clang-tidy? Do you know what rule it is that gets triggered? This function seems rather innocent, perhaps we can just disable the offending rule.

Reply:
The rule is [bugprone-easily-swappable-parameters](https://clang.llvm.org/extra//clang-tidy/checks/bugprone-easily-swappable-parameters.html) I can disable it and remove the `NOLINT` flags.